include/benchmark/benchmark.h

```diff
@@ -164,7 +164,6 @@ BENCHMARK(BM_test)->Unit(benchmark::kMillisecond);
 #ifndef BENCHMARK_BENCHMARK_H_
```

Comment:
could you send a separate PR with the formatting changes? it'll help keep this focused on the actual functional change (and i'm happy to land pure format changes quickly).